Corona-Warn-App/src/test/java/de/rki/coronawarnapp/storage/SubmissionRepositoryTest.kt

```diff
@@ -60,6 +60,9 @@ class SubmissionRepositoryTest : BaseTest() {
     private val testResult = TestResult.PENDING
     private val registrationData = SubmissionService.RegistrationData(registrationToken, testResult)
 
+    private val registrationTokenPreference = mockFlowPreference<String?>(null)
+    private val resultReceivedTimeStamp = Instant.ofEpochMilli(101010101)
```

Comment:
Any specific reason the test value changed from `1` to this?

Reply:
the old version of the initialTestResultReceivedTimestamp was a long that was considered as unset if it was 0, we now store an Instant? that is unset if it is null, the test only checks for null / != null and does not care about the specific value, i decided to have a more specific time than just 1 in the Instant to make it more obvious that its not just a one off error in some places